sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttributeBindingProvider.cs

```diff
@@ -39,6 +39,18 @@ private IBinding TryCreate(BindingProviderContext context)
                 return null;
             }
 
+            // ParameterBindingData is bound by the next binding
+            if (parameter.ParameterType == typeof(ParameterBindingData))
```

Comment:
nit: suggest combining all of these into a single if condition and updating the comment to say that these types are handled by subsequent bindings.